learn/sherif/main_test/src/main/java/frc/robot/CANSparkFunctions.java

```diff
@@ -0,0 +1,8 @@
+package frc.robot;
+
+import com.revrobotics.CANEncoder;
+import edu.wpi.first.wpilibj.SpeedController;
+
+public interface CANSparkFunctions extends SpeedController {
```

Comment:
Calling something `Functions` doesn't really resonate with the terms used in Java or entities containing multiple functions like `Class` or `Interface`.  Maybe call this `CANSparkMaxInterface`